### PR TITLE
feat: add pinning commands

### PR DESCRIPTION
**Pull request by Cafe137**
Adds 3 commands under the group `pinning`:

- `pin`
- `unpin`
- `list`

These commands can be used to pin chunks in retrospect.

There is a known issue at the time of adding this - manifests with index-document cannot be pinned or unpinned. In this case a helpful message is printed.

### CODE REVIEW

**Comment by Cafe137** [comment]
Known issue: when `--index-document` is specified, chunk traversal fails and API call dies with `pin bzz: cannot pin`.

File: test/command/feed.spec.ts [hunk 1]
```diff
@@ -73,4 +73,37 @@ describe('Test Feed command', () => {
...
+
+  it('should print feed using address only', async () => {
+    // create identity
+    await cli({
+      rootCommandClasses,
+      optionParameters,
+      testArguments: ['identity', 'create', '--identity-name', 'test2', '--password', 'test'],
+    })
+    const config = JSON.parse(readFileSync('test/testconfig/feed.config.json').toString())
+    const address = config?.identities?.test2?.wallet?.address
```
Comment (nugaon) [inline test/command/feed.spec.ts:85]: please extract the created address from console.log or from the command's instance (if it stores)

**Review by nugaon — COMMENTED** [review]
Please provide `unpin` tests as well, with checking whether it successfully removed the pinning with `pinning list`

**Review by nugaon — APPROVED** [review]
LGTM!

**Pull request merged**